indico_toolkit/indico_wrapper/workflow.py

```diff
@@ -42,17 +43,24 @@ def get_workflow(self, workflow_id: int) -> Workflow:
         return self.client.call(GetWorkflow(workflow_id))
 
     def submit_documents_to_workflow(
-        self, workflow_id: int, pdf_filepaths: List[str]
+        self,
+        workflow_id: int,
+        pdf_filepaths: List[str],
+        streams: Dict[str, io.BufferedIOBase] = None,
```

Comment:
Couple of improvements I'd like to see as a part of this change:

- Update the existing argument name to be just `files`. It's cleaner, doesn't imply that the files have to be PDFs, and matches the SDK's `WorkflowSubmission` call.
- Since this method can now take either files on disk or byte streams in memory, they should both be optional keyword arguments. Right now `files` is a required positional argument to use `streams`.
- Make the `files` type `PathLike` instead of `str`. This is an ABC that includes `str`, `bytes`, and `Path` objects, all of which are valid ways to specify file paths.

```suggestion
        self,
        workflow_id: int,
        *,
        files: Union[List[PathLike], None] = None,
        streams: Union[Dict[str, io.BufferedIOBase], None] = None,
```